server/url-shorten-service/routes/urlshortener.js

```diff
@@ -30,7 +30,7 @@ module.exports = app => {
 
         shortenedUrlObject = urlRepository.add(shortenedUrlObject);
 
-        return res.status(201).json(shortenedUrlId +"\n");
+        return res.status(201).json(shortenedUrlId);
```

Comment:
Not sure why this is removed. On terminal it is harder to see the result, as no new line is created.

Reply:
What you return is a JSON-String "http://example.com\n" which is not the same than "http://example.com", and thus yields different strings. So something like this `LOGINTOKEN=$(curl --request POST 'http://loadbalancer:80/users/login' --header 'Content-Type: application/json' --data-raw '{"username": "lucien","password":"123password"}')` does not work.